### PR TITLE
move avro serde to suggest, add installation hint

### LINKED ISSUE #21: Move flix-tech/avro-serde-php to optional libraries
Currently flix-tech/avro-serde-php is a hard library requirement. 
Instead it should be part of suggestions (https://getcomposer.org/doc/04-schema.md#suggest). 

Classes that use it may throw descriptive errors if user attempts to use them without it.

### PR DESCRIPTION
**Pull request by nick-zh**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -29,13 +28,17 @@
...
+    "johnkary/phpunit-speedtrap": "^3.1",
+    "flix-tech/avro-serde-php": "^1.3"
   },
   "autoload": {
     "psr-4": {
       "Jobcloud\\Kafka\\": "src/"
     }
   },
+  "suggest": {
+    "flix-tech/avro-serde-php": "Is needed for Avro support"
```
Comment (Steveb-p) [inline composer.json:40]:
```diff
- "flix-tech/avro-serde-php": "Is needed for Avro support"
+ "flix-tech/avro-serde-php": "Required for Avro support"
```

Comment (Steveb-p) [reply composer.json:40]: Is Avro before 1.3 incompatible?

Comment (nick-zh) [reply composer.json:40]: Probably not, no

Comment (TiMESPLiNTER) [reply composer.json:40]: Probably?

Comment (nick-zh) [reply composer.json:40]: No :smile:

**Review by TiMESPLiNTER — APPROVED** [review]

**Pull request merged**